libraries/AP_GPS/AP_GPS.h

```diff
@@ -34,8 +34,6 @@
 // the number of GPS leap seconds
 #define GPS_LEAPSECONDS_MILLIS 18000ULL
 
-#define UNIX_OFFSET_MSEC (17000ULL * 86400ULL + 52 * 10 * MSEC_PER_WEEK - GPS_LEAPSECONDS_MILLIS)
```

Comment:
I confirmed it with Pixhawk.
52 * 10 * MSEC_PER_WEEK The result  was different from the expected value.
52 ULL * 10 ULL * MSEC_PER_WEEK The result  was the expected value.
value:
GPS 1: 578905666865934625 (0x808 AFA 308013121)
GPS 2: 314496000000 (0x49396 C200)
===
Confirmation program
Gcs_send_text_fmt (MAV_SEVERITY_DEBUG, "GPS 1:% llu", 52 * 10 * MSEC_PER_WEEK);
Gcs_send_text_fmt (MAV_SEVERITY_DEBUG, "GPS 2:% llu", 52ULL * 10ULL * MSEC_PER_WEEK);